templates/archives/stories/digital_humanities-teaser.jinja2

```diff
@@ -0,0 +1,11 @@
+{% extends 'archives/story_card.jinja2' %}
+
+{% block story_title %}
+Digital Humanities Through the Ages (or, Just One Age)
+{% endblock %}
+
+{% block story_teaser %}
+A time without digital humanities is better known as a time without computers. Even at its
```

Comment:
I like the second sentence here more, I think it is very compelling to say that despite computer time being rationed, exploration in the humanities was still important.